### PR TITLE
Tie account in system event subject to data and clean up subject names

### PR DESCRIPTION
**Pull request by matthiashanel**
Lookup is a request.
STATSZ was duplicated so it falls into this scheme: 
$SYS.REQ.SERVER.<ID>.<OPERATION> and $SYS.REQ.SERVER.<ID>.PING.<OPERATION>.
The original SYS.REQ.SERVER.<ID>.PING is kept for backwards compatibility. (reason for change in events_test.go)

As for update I didn't go all in on avoiding parsing the jwt twice as it' not a frequent operation and avoids code changes.

There are more changes to come in that area but these where the straight forward ones.
Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

File: server/accounts.go [hunk 1]
```diff
@@ -2937,7 +2937,8 @@ func (dr *DirAccResolver) Start(s *Server) error {
 		}
 	}
 	const accountPackRequest = "$SYS.ACCOUNT.CLAIMS.PACK"
-	const accountLookupRequest = "$SYS.ACCOUNT.*.CLAIMS.LOOKUP"
+	const accountLookupRequest = "$SYS.REQ.ACCOUNT.*.CLAIMS.LOOKUP"
```
Comment (kozlovic) [inline server/accounts.go:2940]: Originally this was used only in this function, but now I wonder if we should move then to events.go (where you have defined number of tokens and index of account).
Also, we could make it a subject with the `%s` token, like `const accountLookupRequest = "$SYS.REQ.ACCOUNT.%s.CLAIMS.LOOKUP"`.
Here, to subscribe you would sprintf with this constant and pass "*" for the "%s".

File: server/accounts.go [hunk 2]
```diff
@@ -3070,7 +3071,7 @@ func (dr *CacheDirAccResolver) Fetch(name string) (string, error) {
 		return "", ErrNoAccountResolver
 	}
 	respC := make(chan []byte, 1)
-	accountLookupRequest := fmt.Sprintf("$SYS.ACCOUNT.%s.CLAIMS.LOOKUP", name)
+	accountLookupRequest := fmt.Sprintf("$SYS.REQ.ACCOUNT.%s.CLAIMS.LOOKUP", name)
```
Comment (kozlovic) [inline server/accounts.go:3074]: Here, I would use a different variable name and sprintf with the constant referenced earlier.

**Review by kozlovic — CHANGES_REQUESTED** [review]
Would make this a constant subject defined in events.go and make use of it to create the subscription on "*" or lookup on specific account.

File: server/accounts.go (hunk 1 above)
Comment (matthiashanel) [reply server/accounts.go:2940]: will make these changes

**Comment by matthiashanel** [comment]
@kozlovic , incorporated your suggestions. also renamed claims.pack as it is a request as well. Moved the constant for it into system events just so we have one place where these can be found.

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**